cmd/okctl/auth/credentials.go

```diff
@@ -0,0 +1,80 @@
+// Package auth deals with credential types towards AWS & Github
+package auth
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/oslokommune/okctl/pkg/config/constant"
+	"github.com/oslokommune/okctl/pkg/context"
+	"github.com/oslokommune/okctl/pkg/okctl"
+)
+
+var (
+	// AwsCredentialsType is the chosen AWS credential type
+	AwsCredentialsType string //nolint:gochecknoglobals
```

Comment:
Question: Should we explain why we choose to add the //nolint:gochecknoglobals?

I think it's okay to ignore warnings in some circumstances, but it would help the future reader of the code to understand why without having to dig through lots of code. It was a little bit hard to find out, because Intellij wasn't able to find any write usages of the variable, because they're being written to via the cobra library in main.go.

Not sure about the best explanation, perhaps: `Using a global variable for easy reference from main.go`.  Of course refereing to stuff in comments like that can get outdated, so I can see the argument for not mentionting this.

Reply:
About them being global: earlier they were defined as globals in package `main`. To move them into the auth package made them slightly more namespaced, and not polluting the global namespace. For me then the next refactoring would be a separate ticket to remove them as global all together.

Reply:
https://trello.com/c/GFjZ8dA1/602-remove-globals-from-cmd-okctl-auth-credentialsgo

Reply:
Alright, sounds good :+1: